tests/acceptance/expected-failures-API-on-OCIS-storage.md

```diff
@@ -51,260 +51,297 @@ Synchronization features like etag propagation, setting mtime and locking files
 -   [apiMain/checksums.feature:374](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiMain/checksums.feature#L374)
 
 #### [Webdav LOCK operations](https://github.com/owncloud/ocis/issues/1284)
--   [apiWebdavLocks/exclusiveLocks.feature:18](https://github.com/owncloud/core/blob/master/tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature#L18)
```

Comment:
The expected-failures file already has all the lines with "-", 3 spaces, then "[". It has been that way for "a long time" (tm). We don't want to change one section of the file to be "-", then 1 space, then "[".

Please revert this indenting change. We can discuss separately if we should change the indent of the whole file, and do that in a separate PR, if we want to.

Reply:
fixed